README.md

```diff
@@ -155,20 +155,34 @@ Type: `Boolean`
 
 If `true`, the modal will receive a `role` of `alertdialog`, instead of its default `dialog`.
 
+### defaultStyles
+
+Type: `Boolean`
+
+By default, styles are applied inline to the  dialog and underlay portions of the component.  However, you can disable all inline styles by setting `defaultStyles` to `false`.  If set, *you must specify all styles externally*, including positioning.  This is helpful if your project uses external CSS assets.
+
+*Note:* `underlayStyle` can still be set inline, but these will be the only styles applied.
+
 ### dialogClass
 
 Type: `String`
 
 Apply a class to the dialog in order to custom-style it.
 
-Be aware that this module does apply various inline styles to the dialog element in order position it.
+Be aware that this module does apply various inline styles to the dialog element in order position it.  To disable *all inline styles*, see `defaultStyles`.
```

Comment:
One space after period. Also, change first sentence to make it clear that this behavior is only *by default*.